snapshot-labs/sx-monorepo
# fix: fix Editor top nav disappearing on small screen when opening the sidebar

### Summary



Closes: #742 

This PR fix the editor top nav being hidden by default top nav on small screen, when opening the sidebar, due to z-index order being lost when using `transform`

### How to test

1. Go to http://localhost:8080/#/s:test.wa0x6e.eth/create on a small screen
2. Open the sidebar
3. The Editor top nav should still be visible


### TODO

- [x] Depends on #792
